utils/scripts/ocm/ocm.py

```diff
@@ -649,327 +717,529 @@ def uninstall_rhoda_addon(self):
+        "--taints",
+        help="Machine pool taints information",
+        action="store",
+        dest="taints",
+        metavar="",
+        default="nvidia.com/gpu=NONE:NoSchedule",
+    )
+    optional_machinepool_cluster_parser.add_argument(
+        "--pool-name",
+        help="Machine pool name",
+        action="store",
+        dest="pool_name",
+        metavar="",
+        default="gpunode",
+    )
+    add_machinepool_parser.set_defaults(func=ocm_obj.add_machine_pool)
+
+    # Argument parsers for uninstall_rhods_addon
+    uninstall_rhods_parser = subparsers.add_parser(
+        "uninstall_rhods_addon",
+        help=("Uninstall rhods addon cluster."),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    required_uninstall_rhods_parser = uninstall_rhods_parser.add_argument_group(
+        "required arguments"
+    )
+
+    required_uninstall_rhods_parser.add_argument(
+        "--cluster-name",
+        help="osd cluster name",
+        action="store",
+        dest="cluster_name",
+        required=True,
+    )
+    uninstall_rhods_parser.set_defaults(func=ocm_obj.uninstall_rhods_addon)
+
+    # Argument parsers for install_rhoda_addon
+    install_rhoda_parser = subparsers.add_parser(
+        "install_rhoda_addon",
+        help=("Install rhoda addon cluster."),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    required_install_rhoda_parser = install_rhoda_parser.add_argument_group(
+        "required arguments"
+    )
+    required_install_rhoda_parser.add_argument(
+        "--cluster-name",
+        help="osd cluster name",
+        action="store",
+        dest="cluster_name",
+        required=True,
+    )
+    install_rhoda_parser.set_defaults(func=ocm_obj.install_rhoda_addon)
+
+    # Argument parsers for uninstall_rhoda_addon
+    uninstall_rhoda_parser = subparsers.add_parser(
+        "uninstall_rhoda_addon",
+        help=("Uninstall rhoda addon cluster."),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    required_uninstall_rhoda_parser = uninstall_rhoda_parser.add_argument_group(
+        "required arguments"
+    )
+    required_uninstall_rhoda_parser.add_argument(
+        "--cluster-name",
+        help="osd cluster name",
+        action="store",
+        dest="cluster_name",
+        required=True,
+    )
+    uninstall_rhoda_parser.set_defaults(func=ocm_obj.uninstall_rhoda_addon)
+    # Argument parsers for create_idp
+    create_idp_parser = subparsers.add_parser(
+        "create_idp",
+        help=("Add an Identity providers to determine how users log into the cluster."),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    optional_create_idp_parser = create_idp_parser._action_groups.pop()
+    required_create_idp_parser = create_idp_parser.add_argument_group(
+        "required arguments"
+    )
+    ldap_create_idp_parser = create_idp_parser.add_argument_group(
+        "  Options for ldap IDP"
```

Comment:
Is the space here intentional?

Reply:
I am not sure, the earlier code had a space here... `black` auto formatted to have a similar visual... it kept the value the same.

Reply:
Removed the space